spec/services/webhooks/events/validation_errors_service_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Webhooks::Events::ValidationErrorsService do
+  subject(:webhook_service) { described_class.new(object: organization, options:) }
+
+  let(:organization) { create(:organization) }
+
+  let(:options) do
+    {
+      errors: {
+        invalid_code: [SecureRandom.uuid],
+        missing_aggregation_property: [SecureRandom.uuid],
+        missing_group_key: [SecureRandom.uuid],
+      }
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.